vars/signArtifacts.groovy

```diff
@@ -156,6 +173,9 @@ void call(Map args = [:]) {
                """
                 }
         }
+        else {
+            error('Unsupported signing platform', args.platform)
```

Comment:
Does `error()` automatically combine the str and the value of `args.platform` into a single line? Or else?

Reply:
Removed this since for promotion and stuff it is using linux as default. To keeping it that way for now.

Reply:
Thanks.